README.md

```diff
@@ -78,7 +78,7 @@ Please make sure you’ve followed the guidelines in
     ```
 
   2. After you have cloned the EarlGrey repository, download all the dependencies using the
-  **setup-earlgrey.sh** (run the script from the cloned repo) script in the `Scripts/` directory.
+  [**setup-earlgrey.sh**](./Scripts/setup-earlgrey.sh) (run the script from the cloned repo) script in the `Scripts/` directory.
```

Comment:
@bootstraponline Why give the path to the file when we're already giving it in the link?